src/main/kotlin/car/Application.kt

```diff
@@ -0,0 +1,18 @@
+package car
+
+import car.business.Cars
+import car.io.InputView
+import car.io.ResultView
+
+fun main() {
+    val inputView = InputView()
+    val input = inputView.input()
+
+    val cars = Cars(input.amount)
+    val resultView = ResultView(cars)
+
+    for (i in 0 until input.tryCount) {
```

Comment:
`repeat()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
    repeat(input.tryCount) {
```

Reply:
횟수 반복을 할 때는 이 코드가 간결하겠군요!! 감사합니다